roles/confluent.variables/defaults/main.yml

```
@@ -719,3 +719,56 @@ kafka_connect_additional_system_admins: "{{rbac_component_additional_system_admi
 # TODO this one honestly does not make sense. You get rolebindings to otherthings and then c3 allows you to view them. Talk to product
 ### List of users to be granted system admin Role Bindings on the Control Center Cluster
 control_center_additional_system_admins: "{{rbac_component_additional_system_admins}}"
+
+# Secrets Protection variables
+
+### Boolean to enable secrets protection on all components.
+secrets_protection_enabled: false
+
+### Full path to configuration file in Kafka broker that contains properties to encrypt. If empty, then secrets protection does not run.
+broker_config_path: ""
```

```suggestion
kafka_broker_secrets_protection_config_path: ""
```

With variable names we have standardized prefixes:

- kafka_broker
- schema_registry
- kafka_connect
- kafka_rest
- ksql
- control_center
- zookeeper

And we want the variables to fully describe what they are (as best as possible). 

So the variable broker_config_path does not describe this has anything to do w secrets protection... it could be mixed up.

Please change the other variables too.

Nice work w the documentation!